samples/js/js-customer-registry/.bundleignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 package-lock.json
 user-function.desc
+.*
```

Comment:
@pvlugter is it ok to exclude a such `.*` files/directories for Javascript?

Reply:
Yes, I think it's fine to exclude all the dot files. If we did want to include a `.gitignore` then we can un-exclude it again. But seems good to me.